documentation/src/main/asciidoc/public/reference/_backend-lucene.adoc

```diff
@@ -670,8 +670,10 @@ you can call `context.availableTokenizers()`, `context.availableTokenizers()`
 and `context.availableTokenFilters()` on the context passed to your analysis configurer;
 this will return a set of all valid names.
 
-To know more about the behavior of these character filters, tokenizers and token filters,
-either browse the link:{luceneJavadocUrl}[Lucene Javadoc] or read the corresponding section on the
+To learn more about the behavior of these character filters, tokenizers and token filters,
+either browse the link:{luceneJavadocUrl}[Lucene Javadoc],
+in particular, look through various packages of link:{luceneAnalysisCommonJavadocUrl}[common analysis components],
+or read the corresponding section on the
```

Comment:
There are other links to Lucene javadoc in the documentation that you might want to update; CTRL+F should allow you to find them.